.github/ISSUE_TEMPLATE/kel-issue-template.md

```diff
@@ -0,0 +1,29 @@
+---
+name: KEL Issue Template
+about: 'Testing out template to be used for KEL issues. '
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Summary
+
+### Impact 
+_Please include hard deadlines, if the exhibit is part of an event, the issue is stopping work, etc._
+
+### Priority recommendation
+
+- [ ] asap
+- [ ] within the next 3 weeks
+- [ ] PO will prioritize
+
+## Sudden Priority Justification
```

Comment:
This header's only required if asap or "within the next 3 weeks" is checked.

Reply:
Would you like to keep that header in (with descrip), or would you prefer to add it if needed?

Reply:
Whatever you'd prefer, I thought you might just want a note so you don't feel like you have to always add it.

Reply:
Ill keep it with the descrip. it is really helpful